examples/viewer.py

```diff
@@ -782,6 +845,9 @@ def print_help_text(self) -> None:
     't':        Load URDF from filepath
                 (+SHIFT) quick re-load the previously specified URDF
                 (+ALT) load the URDF with fixed base
+    'o':        Add a random object to the scene in front of the agent.
```

Comment:
For future reference: it would be nice if the python viewer's UI didn't become an explosion of hotkeys as happened to the C++ viewer. It seems to me the root of the problem is that our K different viewer use cases all have to fight over the same set of physical keys.